tensorflow/python/data/experimental/ops/io.py

```diff
@@ -64,14 +66,14 @@ def save(dataset, path, compression=None, shard_func=None):
   dataset = make_dataset()
   def custom_shard_func(element):
     return 0
-  dataset = tf.data.experimental.save(
+  dataset = tf.data.save(
```

Comment:
`save` does not return anything (this is a pre-existing issue but please fix it)